src/components/App/App.tsx

```diff
@@ -1,33 +1,64 @@
 import * as React from 'react';
 import { Content, Header, Tabs } from 'src/components';
-import { HARD_DATA } from 'src/constants';
+import { CHUNKS_API, HARD_DATA, STUDENT_TABS } from 'src/constants';
+import { IChunk } from 'src/types';
 import * as style from './App.scss';
 
 interface IState {
+  chunks: Map<string, IChunk>;
   selectedTabId: number;
 }
 
 export class App extends React.Component<{}, IState> {
   public state = {
+    chunks: new Map(Object.entries(HARD_DATA)),
     selectedTabId: 0,
   };
-  public handleClick = (index: number) => {
+  public handleClick = (id: number) => {
     this.setState({
-      selectedTabId: index,
+      selectedTabId: id,
     });
   };
+
+  public componentDidMount() {
+    this.loadChunks();
+  }
+
   public render() {
-    const { selectedTabId } = this.state;
+    const { selectedTabId, chunks } = this.state;
     return (
       <div className={style.app}>
         <Header />
         <Tabs
-          tabs={HARD_DATA}
+          tabs={STUDENT_TABS}
           selectedTabId={selectedTabId}
           handleClick={this.handleClick}
         />
-        <Content selectedChunks={HARD_DATA[selectedTabId]} />
+        <Content
+          selectedChunks={STUDENT_TABS[selectedTabId].chunks.map(
+            chunk => chunks.get(chunk) as IChunk
+          )}
+        />
       </div>
     );
   }
+
+  private async loadChunks() {
+    const chunksJson = await (await fetch(CHUNKS_API)).json();
```

Comment:
The chunks api have severald pages so it isn't nessesarly enough to only load the first page.

Reply:
Might be better to add some filtering to the chunks api allowing you to only select the chunks you want using the 'key' parameter.

Reply:
Since we load ~15-20 chunks I think we're better off loading the whole resource and filter it locally rather than sending 20 HTTP requests for the content. I don't think it matters *much*, though.